src/server/page.tsx

```diff
@@ -87,7 +87,10 @@ const template = ({
     }
     <div id="react-root">${body}</div>
       <script nonce="${nonce}">
-      window.__INITIAL_STATE__ = ${JSON.stringify(initialState)};
+      window.__INITIAL_STATE__ = ${JSON.stringify(initialState).replace(
```

Comment:
Ah! This fixes the issue with the hidden blank space that broke the page right?

Reply:
Yes!